index.html

```diff
@@ -138,6 +138,7 @@ <h4 class="title-1">Multi-Post Stories<br>Gain+Glory</h4>
            <button class="button active-btn" type="button">See Project</button>
           </div>
         </div>
+        
```

Comment:
- [ ] Kindly **As suggested by the previous reviewer,** try to replicate the Figma template design by styling and following the alignments as given 🙏

**currently 👇**
https://www.loom.com/share/7e16dddd83874dc7b26c2d8b6e3db795

I think it would be better to prevent scrolling horizontally in your CSS to be able to align your webpage to replicate the Figma design 👍 👍 

example:
```
body {
  overflow-x: hidden;
  width: 100%;
}
```
_you may kindly follow this[ link ](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp)for more info on hidden overflow_ 

_You may also follow this [link](https://www.geeksforgeeks.org/css-align/) on CSS alignment 😃😎_